### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Kitsunet2** [vrchat-community/creator-companion#500]
### Describe the bug

the creator companion app is stuck in white screen everytime i open it, i tried some methods like changing names of the config, reinstalling edge and web view, unistalling and installing again, restart the pc but nothing works, i put the last log file and a screen 
[20240625T185405-4111c70137454153a5b74050825c67d7-log.txt](https://github.com/user-attachments/files/15980894/20240625T185405-4111c70137454153a5b74050825c67d7-log.txt)

### To Reproduce

- open the CC
-give permision as administrator
-opens but stuck in white screen

### Expected behavior

-open an works

### Screenshots

![imagen_2024-06-25_185423287](https://github.com/vrchat-community/creator-companion/assets/173762537/9486a2a2-645b-48b6-b320-117738b7f7c1)

### Related Product

VCC (VRChat Creator Companion)

### Product Version

2.3.1

### OS

Windows 11

**Comment by anatawa12** [comment]
Is there any information on the console tab of devtools, which can be opened with F12 key?

**Comment by Kitsunet2** [comment]
I open the program as administrator again, press f12 but nothing happends, remains stuck in white screen, i need to do something apart o activate something?

**Comment by Kitsunet2** [comment]
idk is this could help but i put my settings file cuz i see that the problems maybe are related to that (every time that i delete this file reload like this)
[settings.json](https://github.com/user-attachments/files/15984215/settings.json)

**Comment by anatawa12** [comment]
How about ctrl + shift+i?

**Comment by orels1** [comment]
Hey!
Since the devtools aren't opening for you, I have a feeling you might be missing the webview runtime.
You can try installing it using the "Evergreen Bootstrapper" link you can get here:

https://developer.microsoft.com/en-us/microsoft-edge/webview2/?form=MA13LH#download

![image](https://github.com/user-attachments/assets/a195fe27-e690-4618-aefa-088292587cc9)

**Comment by Kitsunet2** [comment]
srry for  the late response, i was busy these days, and i tried to just download, run the exe and says this: 
![image](https://github.com/user-attachments/assets/025303c1-d00c-4c7b-8147-8d848d466c0a)

i need to do something more before just run the exe installer?

**Comment by orels1** [comment]
hmm, interesting
Are you able to open the VCC and then open `http://localhost:5476/` in your regular browser?

**Comment by Kitsunet2** [comment]
yes, i could open the page in the browser while the VCC is freeze in the white screen
